fixtures/proc-macro/src/proc-macro.udl

```diff
@@ -1,7 +1,6 @@
-namespace uniffi_proc_macro {};
+namespace uniffi_proc_macro { };
 
-// Test that redundant definition of a dictionary works
-dictionary Two {
-    string a;
-    sequence<boolean>? b;
+// Use this to test that types defined in the UDL can be used in the proc-macros
+dictionary Zero {
+    string inner;
```

Comment:
With this change, you can no longer provide a redundant/duplicate definition between the UDL and proc-macros, but hopefully the story is still good enough:
  - You can define a type in the UDL and use it in the proc-macros, like in this test.
  - You can't `[uniffi:export]` a type in the proc-macro and use it in the UDL.

I think this should be okay for the transition period, especially since proc-macros support a lot of features at this point.  If there are issues, I can add code to handle the second case, but I'm hoping to avoid that.